core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1296,6 +1296,7 @@ abstract class RDD[T: ClassTag](
    * an exception if called on an RDD of `Nothing` or `Null`.
    */
   def take(num: Int): Array[T] = withScope {
+    val takeRampUpRate = conf.getInt("spark.rdd.take.rampUpRate", 4)
```

Comment:
Also would someone really want to set this differently for RDD vs other things? Trying to avoid proliferation of config. Would someone reasonably ever tune this any way?